draft-ietf-rats-architecture.md

```diff
@@ -1424,30 +1425,57 @@ in terms of `time(RX_v)-time(RG_v)`, then the Relying Party can check
-As any other role, the Handle Distributor role can be taken on by a dedicated entity or collapsed with other roles, such as a Verifier.
-The use of handles can compensate for a lack of clocks or other sources of time on entities taking on RATS roles.
-The only entity that requires access to a source of time is the entity taking on the role of Handle Distributor.
+A third option to establish the epoch of Evidence or Attestation Result
+generation - next to nonces and timestamps ({{freshness}}) - is by using
+"handles".
 
-Handles are different from nonces as they can be used more than once and can be used by more than one entity at the same time.
-Handles are different from timestamps as they do not have to convey information about a point in time, but their reception creates that information.
-The reception of a handle is similar to the event that increments a relative tickcounter.
-Receipt of a new handle invalidates a previously received handle.
+Handles are opaque, non-predictable data regularly broadcasted to all protocol
+participants by a Handle Distributor.  Each handle represents a specific epoch,
+in which generated Evidence can be asserted to be fresh.
 
-In this example, Evidence generation based on received handles always uses the current (most recent) handle.
-As handles are distributed over the network, all involved entities receive a fresh handle at roughly the same time.
-Due to distribution over the network, there is some jitter with respect to the time the Handle is received, time(HR), for each involved entity.
-To compensate for this jitter, there is a small period of overlap (a specified offset) in which both a current handle and corresponding former handle are valid in Evidence appraisal: `validity-duration = time(HR'_v) + offset - time(HR_v)`. The offset is typically based on a network's round trip time.
-Analogously, the generation of valid Evidence is only possible, if the age of the handle used is lower than the validity-duration: `time(HR_v) - time(EG_a) < validity-duration`.
+The distribution of a new handle determines the transition from one epoch to
+the next.  The transition between epochs goes through an "overlap" phase in
+which the previous epoch is still extant while the current epoch is
+established.  The span of the overlap phase is determined solely by the
+transmission delay of the underlying communication channel,
+`time(HR)-time(HT)`.  Once the overlap phase is over, an epoch is fully
+established.
 
-From the point of view of a Verifier, the generation of valid Evidence is only possible, if the age of the handle used in the Evidence generation is younger than the duration of the distribution interval -- "(time(HR'_v)-time(HR_v)) - (time(HR_a)-time(EG_a)) \< validity-duration".
+In order to determine freshness of Evidence or Attestation Results, consumers
+of messages based on handles need to understand whether an epoch is fully
+established or in overlap phase.
```

Comment:
This is confusing.  Remove it?

Reply:
Can you please explain where is the confusion?

We think that being able to distinguish between a fully established epoch and one that is still in overlap phase is crucial for a receiver that needs to decide whether to accept or refuse an incoming message.

Reply:
I don't understand your point.   If it's in overlap phase, it should still be accepted.

Reply:
Sure.  However, in order to be able to _reject_ a message coming from the previous epoch a receiver needs to be able to say: _"overlap is over and the new epoch is now fully established, therefore I will assume whatever comes from a previous epoch is a replay"_

Reply:
Accepting an incoming Conceptual Message without overlap phase awareness should be very strongly discouraged, as it can break the whole RATS concept. I understand that it is not obvious or easy for a receiver to infer that it is in an overlap phase. Nevertheless, delimiting factors should be illustrated in this context as best as we can.

Reply:
It was pointed out to me that my comment was mislead. Unfortunately, I have to agree.

What I intended to convey was: yes, of course the message is accepted, but blindly accepting it without awareness of an overlap phase poses a security risk in the following appraisal procedures due to lack of relevant context information. IHTH